testscommon/goroutines/testdata/test3.data

```diff
@@ -0,0 +1,85 @@
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).insert(0xc0004074a0, {{0xc000b817f0, 0x7f8b54309000, 0x7f8b54305700}, 0xc00040ed00}, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).insert(0xc000406a20, {{0xc000b817f0, 0x4440a5, 0xc000793ba8}, 0xc00040e600}, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).insert(0xc0007b8180, {{0xc000b817f0, 0x4474c6, 0x2000}, 0xc0007b7410}, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).insert(0xc000406420, {{0xc000b817f0, 0xc000b86660, 0x4474c6}, 0xc00040e200}, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).insert(0xc0004061e0, {{0xc000b817f0, 0x10, 0xc000b817d0}, 0x10}, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:264 +0x25b
+github.com/libp2p/go-cidranger.(*prefixTrie).Insert(0xc0004061e0, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/trie.go:95 +0xdb
+github.com/libp2p/go-cidranger.(*versionedRanger).Insert(0x1005e0d, {0x130a8a0, 0xc0002a6640})
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-cidranger@v1.1.0/version.go:29 +0xd4
+github.com/libp2p/go-libp2p-asn-util.newAsnStore()
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-libp2p-asn-util@v0.0.0-20200825225859-85005c6cf052/asn.go:61 +0x186
+github.com/libp2p/go-libp2p-asn-util.newIndirectAsnStore.func1()
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-libp2p-asn-util@v0.0.0-20200825225859-85005c6cf052/asn.go:89 +0x65
+created by github.com/libp2p/go-libp2p-asn-util.newIndirectAsnStore
+	/home/user/Dev3/go/pkg/mod/github.com/libp2p/go-libp2p-asn-util@v0.0.0-20200825225859-85005c6cf052/asn.go:87 +0xa5
+
+goroutine 53 [select]:
+github.com/ElrondNetwork/elrond-go/trie.(*trieStorageManager).doProcessLoop(0xc0000f0000, {0x13248d8, 0xc000753100}, {0x1315fc8, 0x1ca7ca0}, {0x1315f98, 0xc0003993f8})
+	/home/user/Dev3/go/src/github.com/ElrondNetwork/elrond-go/trie/trieStorageManager.go:110 +0x11f
+github.com/ElrondNetwork/elrond-go/trie.(*trieStorageManager).doCheckpointsAndSnapshots(0xc000103040, {0x13248d8, 0xc000753100}, {0x1315fc8, 0x1ca7ca0}, {0x1315f98, 0xc0003993f8})
+	/home/user/Dev3/go/src/github.com/ElrondNetwork/elrond-go/trie/trieStorageManager.go:104 +0x3f
+created by github.com/ElrondNetwork/elrond-go/trie.NewTrieStorageManager
+	/home/user/Dev3/go/src/github.com/ElrondNetwork/elrond-go/trie/trieStorageManager.go:99 +0x5f7
```

Comment:
empty line at the end of file? Also, can you remove `Dev3` from test data paths?